gradle.properties

```diff
@@ -0,0 +1,27 @@
+pluginGroup = com.finanteq.plugins.idea.cucumber-kotlin-idea-plugin
```

Comment:
can we have it just hardcoded in `build.gradle`? Keeping them here and loading via `providers.gradleProperty("pluginGroup").get()` is unnecessary complication which requires additional roundtrips to get real values